pilot/pkg/model/authentication.go

```diff
@@ -214,6 +214,20 @@ func (policy *AuthenticationPolicies) GetPeerAuthenticationsForWorkload(namespac
 	return getConfigsForWorkload(policy.peerAuthentications, policy.rootNamespace, namespace, workloadLabels)
 }
 
+// GetPeerAuthenticationsForNamespace returns a list of peer authentication policies matching the namespace or in the root namespace.
```

Comment:
@howardjohn is it overkill to add all the PeerAuthn to dependent configs? Should we just say any PeerAuthn invalidates the cache?

We'll probably want to grab them at some point anyway so that we can later match each endpoint to the selector for the tlsMode override we discussed, but for XDS cache idk if this is worth it.

Reply:
Any PeerAuthn changes in the namespace or root namespace will cause this to change. I can probably follow up with something to narrow it down to WorkloadSelector.

Reply:
removed... also can't reply to your comment about port-level DR policy – that is handled now but in a different place.